package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "acorn": "^5.0.3",
+    "acorn-dynamic-import-with-walk-support": "^2.1.0",
```

Comment:
It looks like your PR is about to be merged into acorn-dynamic-import proper. I'd prefer to use the unforked version.